src/Files.App/Actions/Content/Background/SetAsSlideshowBackgroundAction.cs

```diff
@@ -18,11 +20,14 @@ public override RichGlyph Glyph
 
 		public override bool IsExecutable =>
 			base.IsExecutable &&
-			context.SelectedItems.Count > 1;
+			ContentPageContext.SelectedItems.Count > 1;
 
 		public override Task ExecuteAsync(object? parameter = null)
 		{
-			var paths = context.SelectedItems.Select(item => item.ItemPath).ToArray();
+			if (ContentPageContext.SelectedItems.Count <= 1)
```

Comment:
Is there a scenario where this is ever true?

Reply:
I dont know, just doing the same as other actions. Like, checking if `if (ContentPageContext.SelectedItem is null)` is true (while IsExecutable includes this checking).

Reply:
Done.